### PR TITLE
Mobile view

### PR DESCRIPTION
**Pull request by jhack32**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -20,7 +20,7 @@
     <a class="item portfolio" href="#portfolio">portfolio</a>
     <a class="item services" href="#services">services</a>
     <a class="item team" href="#team">team</a>
-    <a class="right item">contact</div>
+    <a class="right item" href="mailto:Cortlandtlabs@gmail.com">contact</div>
```
Comment (dijonkitchen) [inline index.html:23]: Not sure we want mail yet. Might get spam.

Comment (jhack32) [reply index.html:23]: This is the contact you referred to in the issue, or were you talking about something else.

EDIT: Just re-read the issue, you said a contact button. But as for spam mail, don't think it's an issue

Comment (dijonkitchen) [reply index.html:23]: Nope, I meant the contact button/tab wasn't viewable on mobile.

File: css/mobile.css [hunk 2]
```diff
@@ -21,22 +21,18 @@
...
       background-position: center;
+      margin: 0 auto;
     }
-    .mission-statement.mobile {
-      display: block;
+    .mission-statement {
       position: relative;
       font-size: 4.5em;
+      line-height: 1em;
+      padding-top: 33vh;
```
Comment (aattsai) [inline css/mobile.css:15]: To fit all components in firstSegment in mobile, I think under width 450px, mission statement needs padding-top 18vh

**Comment by aattsai** [comment]
Do you want to remove all the excess code in mobile.css?

**Comment by jhack32** [comment]
Done

**Comment by aattsai** [comment]
👍

**Pull request merged**